### PR TITLE
Fixes #34381 - Change example for --order

### PR DESCRIPTION
**Pull request by adiabramovitch**

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
@ofedoren do you wanna take this one? :)

**Comment by ofedoren** [comment]
@ezr-ondrej, of course, forgot to assign myself.

**Review by ofedoren — APPROVED** [review]
Thanks, @adiabramovitch! LGTM.

Could you please open a CP against 3.1-stable?

**Comment by ofedoren** [comment]
CP 3.1: 9d3e3d3fdeb5de8dfeb2d80b456ab60ab2ec9121

File: app/controllers/api/v2/base_controller.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ class BaseController < Api::BaseController
 
       def_param_group :search_and_pagination do
         param :search, String, :desc => N_("filter results")
-        param :order, String, :desc => N_("Sort and order by a searchable field, e.g. 'id DESC'")
+        param :order, String, :desc => N_("Sort and order by a searchable field, e.g. '<field> DESC'")
```
Comment (ezr-ondrej) [inline app/controllers/api/v2/base_controller.rb:21]: @ofedoren, @adiabramovitch I believe this is parsed by Maruku and it complains quite loudly:
```
| Maruku tells you:
+---------------------------------------------------------------------------
| Maruku cannot parse this block of HTML/XML:
| |<field> DESC'
| Missing end tag for 'field' (got 'html')
| Line: 4
| Position: 172
| Last 80 unconsumed characters:
+---------------------------------------------------------------------------
!/home/oezr/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/maruku-0.7.3/lib/maruku/helpers.rb:72:in `rescue in md_html'
!/home/oezr/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/maruku-0.7.3/lib/maruku/helpers.rb:67:in `md_html'
!/home/oezr/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/maruku-0.7.3/lib/maruku/input/parse_span.rb:513:in `read_inline_html'
!/home/oezr/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/maruku-0.7.3/lib/maruku/input/parse_span.rb:115:in `read_span'
\___________________________________________________________________________

Wrapping bad html in a PRE with class 'markdown-html-error'
|<field> DESC'
```

Comment (ofedoren) [reply app/controllers/api/v2/base_controller.rb:21]: Well, we have bunch of these already anyway :/

Comment (ezr-ondrej) [reply app/controllers/api/v2/base_controller.rb:21]: well... this is the only one I have in my develop setup currently.
Isn't there a way to escape this for Maruku not to read it as HTML tag?

Comment (ofedoren) [reply app/controllers/api/v2/base_controller.rb:21]: Honestly, didn't look at this yet :/ We have a redmine issue [1] to track such warnings.

[1] - https://projects.theforeman.org/issues/31889

**Pull request merged**